core/lib/dal/migrations/20231213192041_applied-snapshot-status.up.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE applied_snapshot_status
```

Comment:
Bikeshedding: Maybe name this `snapshot_recovery`? "Status" sound like something logically small / confined, while I'm not sure that this entity will be this.